rllib/models/torch/modules/noisy_layer.py

```diff
@@ -16,7 +17,11 @@ class NoisyLayer(nn.Module):
     vanish along the training procedure
     """
 
-    def __init__(self, in_size, out_size, sigma0, activation="relu"):
+    def __init__(self,
+                 in_size: int,
+                 out_size: int,
+                 sigma0: float,
+                 activation: str = "relu"):
```

Comment:
Sorry, the incomplete docstring here is on me. Could you complete it, though? Thanks.